test/e2e-angular.sh

```diff
@@ -28,6 +28,10 @@ echo ""
 echo "Angular App"
 cd $NG_EXAMPLE
 killWebserver 4200
+npm install
+sleep 5
+npm run build
```

Comment:
why install packages and npm run build here. In the pipeline this should already be done before this script runs altogether

Reply:
Without these commands "Integration Tests" are failing